### ISSUE TITLE
account limitation

### ISSUE THREAD

**Issue by Prabh-Saini** [Prabh-Saini/inconspicuousname#34]
due to the useragent list there is a maximum of 3 accounts useable

**Comment by Prabh-Saini** [comment]
```
Traceback (most recent call last):
  File "C:\Users\REDACTED\PycharmProjects\pythonProject\main.py", line 1088, in <module>
    main(mainuserid)
  File "C:\Users\REDACTED\PycharmProjects\pythonProject\main.py", line 1054, in main
    desktop()
  File "C:\Users\REDACTED\PycharmProjects\pythonProject\main.py", line 906, in desktop
    b = create_b_instance(mobile_instance=False)
  File "C:\Users\REDACTED\PycharmProjects\pythonProject\main.py", line 94, in create_b_instance
    options.add_argument(f"user-agent={desktop[randint(1, 3)]}")
IndexError: list index out of range
```
bro math not be fire

**Comment by Prabh-Saini** [comment]
fixed in #39 i think
